src/azure-cli/azure/cli/command_modules/mysql/tests/latest/test_mysql_scenario.py

```diff
@@ -94,6 +95,7 @@ def test_mysql_flexible_server_mgmt(self, resource_group):
     
     @AllowLargeResponse()
     @ResourceGroupPreparer(location=DEFAULT_LOCATION)
+    @unittest.skip("Currently blocked due to creation of 'Azure Database for MySQL - Single Server' no longer supported on March 19 2024.")
     def test_mysql_flexible_server_import_create(self, resource_group):
```

Comment:
@honghr Can you pls review this bump version PR as there're so many mysql tests which can't pass

Reply:
@nasc17  I know that we will retire single server but not now and these migration commands also need to help customer to migrate to flexible server. So we need keep these test can be executed successfully.

Reply:
@honghr this test starts with the creation of a single server, which is no longer supported through CLI. Can you please advise how best to go about performing this check since the first step causes failure?

Reply:
@evelyn-ys and @nasc17  Let's skip this case now.